receiver/tcplogreceiver/README.md

```diff
@@ -0,0 +1,44 @@
+# TCP Receiver
+
+Receives logs from tcp using
+the [opentelemetry-log-collection](https://github.com/open-telemetry/opentelemetry-log-collection) library.
+
+Supported pipeline types: logs
+
+> :construction: This receiver is in alpha and configuration fields are subject to change.
+
+## Configuration
+
+| Field             | Default          | Description                                                                                        |
+| ---               | ---              | ---                                                                                                |
+| `output`          | Next in pipeline | The connected operator(s) that will receive all outbound entries                                   |
+| `max_buffer_size` | `1024kib`        | Maximum size of buffer that may be allocated while reading TCP input                               |
+| `listen_address`  | required         | A listen address of the form `<ip>:<port>`                                                         |
+| `tls`             | nil              | An optional `TLS` configuration (see the TLS configuration section)                                |
+| `write_to`        | $                | The body [field](/docs/types/field.md) written to when creating a new log entry                    |
+| `attributes`      | {}               | A map of `key: value` pairs to add to the entry's attributes                                       |
```

Comment:
We have `labels` in `filelog. Let's make sure we are consistent. I suggest we use `attributes` everywhere.

Reply:
I've created a PR to update terminology in `filelog`: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/3214